Src/FluentAssertions/FluentAssertions.csproj

```diff
@@ -40,58 +43,82 @@
       <_Parameter1>Benchmarks, PublicKey=00240000048000009400000006020000002400005253413100040000010001002d25ff515c85b13ba08f61d466cff5d80a7f28ba197bbf8796085213e7a3406f970d2a4874932fed35db546e89af2da88c194bf1b7f7ac70de7988c78406f7629c547283061282a825616eb7eb48a9514a7570942936020a9bb37dca9ff60b778309900851575614491c6d25018fadb75828f4c7a17bf2d7dc86e7b6eafc5d8f</_Parameter1>
     </AssemblyAttribute>
   </ItemGroup>
-  <ItemGroup>
-    <None Remove="BannedSymbols.txt" />
```

Comment:
I can't recall why I added this.
Does removing it changes anything?
I.e. is it excluded from the nuget package and does the analyzer still flag usage of banned APIs?